db/migrate/20170120154842_add_started_on_column_to_miq_tasks.rb

```diff
@@ -0,0 +1,5 @@
+class AddStartedOnColumnToMiqTasks < ActiveRecord::Migration[5.0]
```

Comment:
@gtanzillo I've added callback ```after_update_commit``` to ```Job``` which will take care of synchronization for ```started_on``` column between ```Job``` and ```MiqTask```